region/brinstar/red/Alpha Power Bomb Room.json

```diff
@@ -208,6 +208,122 @@
         ]}
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "G-Mode Morph Power Bomb the Blocks",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphPowerBomb",
+        {"or": [
+          "h_canArtificialMorphMovement",
+          {"and": [
+            "h_canArtificialMorphBombs",
+            "canBePatient"
+          ]}
+        ]}
+      ],
+      "note": [
+        "Traverse the room while avoiding or killing the Boyons.",
+        "If the Chozo item is still there, it is best to exit G-Mode before the final Power Bomb goes off to ensure that PLMs aren't overloaded.",
+        "If Samus falls into a Samus Eater, she will be teleported into it when exiting G-Mode."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "G-Mode Morph Power Bomb the Blocks (Power Bombs Only)",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphPowerBomb",
+        {"or": [
+          {"ammo": { "type": "PowerBomb", "count": 6}},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 4}},
+            "h_canArtificialMorphBombHorizontally"
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 4}},
+            "canTrickyJump",
+            "canHitbox"
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 3}},
+            "h_canArtificialMorphBombHorizontally",
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 1
+            }}
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 2}},
```

Comment:
How does this one work?

Reply:
This is the 4 PB variant (2 here, 1 at the top, 1 on the next line)
horizontally boost from the runway through the boyons onto small platform 2) boost to 3) hitbox through the boyons to 4) and then 1 more for the wall.

Reply:
Ah, I wasn't reading h_canAriticialMorphBombHorizontally as adding a power bomb